[2025-07-29T01:20:58.361+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-29T01:20:58.381+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup manual__2025-07-28T23:21:04.451933+00:00 [queued]>
[2025-07-29T01:20:58.387+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fraud_detection_training.cleanup manual__2025-07-28T23:21:04.451933+00:00 [queued]>
[2025-07-29T01:20:58.387+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-29T01:20:58.395+0000] {taskinstance.py:2890} INFO - Executing <Task(BashOperator): cleanup> on 2025-07-28 23:21:04.451933+00:00
[2025-07-29T01:20:58.401+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=112) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-29T01:20:58.402+0000] {standard_task_runner.py:72} INFO - Started process 113 to run task
[2025-07-29T01:20:58.403+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'fraud_detection_training', 'cleanup', 'manual__2025-07-28T23:21:04.451933+00:00', '--job-id', '24', '--raw', '--subdir', 'DAGS_FOLDER/fraud_detection_training_dag.py', '--cfg-path', '/tmp/tmpe8at2mbt']
[2025-07-29T01:20:58.404+0000] {standard_task_runner.py:105} INFO - Job 24: Subtask cleanup
[2025-07-29T01:20:58.447+0000] {task_command.py:467} INFO - Running <TaskInstance: fraud_detection_training.cleanup manual__2025-07-28T23:21:04.451933+00:00 [running]> on host b2497d67dac2
[2025-07-29T01:20:58.514+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='kanewyp' AIRFLOW_CTX_DAG_ID='fraud_detection_training' AIRFLOW_CTX_TASK_ID='cleanup' AIRFLOW_CTX_EXECUTION_DATE='2025-07-28T23:21:04.451933+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-28T23:21:04.451933+00:00'
[2025-07-29T01:20:58.515+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-29T01:20:58.526+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-07-29T01:20:58.527+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '\n        echo "Cleaning up resources..."\n        rm -f /app/tmp/*.pkl\n        echo "Cleanup completed."\n        ']
[2025-07-29T01:20:58.538+0000] {subprocess.py:99} INFO - Output:
[2025-07-29T01:20:58.540+0000] {subprocess.py:106} INFO - Cleaning up resources...
[2025-07-29T01:20:58.544+0000] {subprocess.py:106} INFO - Cleanup completed.
[2025-07-29T01:20:58.544+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-07-29T01:20:58.566+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-29T01:20:58.566+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=fraud_detection_training, task_id=cleanup, run_id=manual__2025-07-28T23:21:04.451933+00:00, execution_date=20250728T232104, start_date=20250729T012058, end_date=20250729T012058
[2025-07-29T01:20:58.617+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-29T01:20:58.636+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-29T01:20:58.637+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
